src/vector/index.html

```diff
@@ -37,8 +37,19 @@
   <body style="height: 100%;">
     <section id="matrixchat" style="height: 100%;"></section>
     <noscript>Sorry, Riot requires JavaScript to be enabled.</noscript>
-    <% for (var i=0; i < htmlWebpackPlugin.files.js.length; i++) {%>
-       <script src="<%= htmlWebpackPlugin.files.js[i] %>"></script>
+    <% for (var i=0; i < htmlWebpackPlugin.files.js.length; i++) {
+        // Not a particularly graceful way of not putting the indexeddb worker script
+        // into the main page
+        if (htmlWebpackPlugin.files.js[i].endsWith('indexeddb-worker.js')) {
```

Comment:
I'm fairly sure Babel doesn't get applied to this file. If it doesn't, then you cannot use `endsWith` as that is ES6 only :( https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith